code/game/machinery/doors/multi_tile.dm

```diff
@@ -7,10 +7,14 @@
 /obj/machinery/door/airlock/multi_tile/Initialize()
 	. = ..()
 	SetBounds()
+	setup_hatch()
+	update_icon()
 
 /obj/machinery/door/airlock/multi_tile/Move()
 	. = ..()
 	SetBounds()
+	setup_hatch()
+	update_icon()
```

Comment:
Not so sure on this, but these doors are uncommon enough that it won't be a problem.